cgmanifest.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Registrations": [
```

Comment:
Do we need to also register other components listed in the ThirdPartyNotices?

Reply:
Something like PLCR? I guess we can register native SDK to avoid listening all this stuff

Reply:
Yes, I think we have to. Even in the Apple SDK we list PLCR dependencies like Protobuf. Unless you can find a guidance that proves otherwise.

Reply:
Anyways the missing existing dependencies is probably not something we need to fix now in this feature. They are already listed in the ThirdPartyNotice, we can add the tracking via Component Governance in another PR.

Reply:
Also I was thinking if we are shipping this binary in our package we'll probably need to add its license in the Notice.